src/guides/v2.4/install-gde/trouble/troubleshooting.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
I don't think we should manually index the KB on devdocs like this. I think we should remove this topic.